src/channel.cpp

```diff
@@ -578,39 +578,39 @@ void Channel::loadChannel(const json_t* channode) {
 
 string Channel::modeToString() {
     switch (chatMode) {
-        default:
         case CMM_BOTH:
             return "both";
         case CMM_ADS_ONLY:
             return "ads";
         case CMM_CHAT_ONLY:
             return "chat";
+        // For now, explicitly defining the default as both.
+        default:
+            return "both";
```

Comment:
Defaults look better at the bottom, yes, but in the interest of reducing duplication, please group things that do the same.

Reply:
Agreed, this was more of a potential explicit definition (essentially replacing what used to be the explicit return statement directly below this with the switch equivalent). Happy to revert and simply leave the return statement removed and the default grouped with case CMM_BOTH.